Documentation/ceph-upgrade.md

```diff
@@ -51,23 +52,23 @@ With this upgrade guide, there are a few notes to consider:
 
 Unless otherwise noted due to extenuating requirements, upgrades from one patch release of Rook to
 another are as simple as updating the common resources and the image of the Rook operator. For
-example, when Rook v1.5.3 is released, the process of updating from v1.5.0 is as simple as running
+example, when Rook v1.6.1 is released, the process of updating from v1.6.0 is as simple as running
 the following:
 
-First get the latest common resources manifests that contain the latest changes for Rook v1.5.
+First get the latest common resources manifests that contain the latest changes for Rook v1.6.
 ```sh
-git clone --single-branch --branch v1.5.3 https://github.com/rook/rook.git
+git clone --single-branch --branch v1.6.0 https://github.com/rook/rook.git
```

Comment:
The example branch tag should be the same as the target version
```suggestion
git clone --single-branch --branch v1.6.1 https://github.com/rook/rook.git
```

Reply:
I think adding `--depth 1` is interesting too.

Reply:
Since we're using `--single-branch`, that's already limiting it, right? Just in the interest of keeping the command simple.

Reply:
Yes and no, we still pull the entire history of the branch.

Reply:
`git clone --depth 1 --branch v1.6.0 https://github.com/rook/rook.git` is enough. It's fast and simple.

```man 1 git-clone
       --depth <depth>
           Create a shallow clone with a history truncated to the specified number of commits. Implies --single-branch unless --no-single-branch is given to fetch the histories near the tips of all branches. If you want to clone submodules shallowly, also pass --shallow-submodules.
```